From: Christoph Hellwig <hch@lst.de>
Subject: kernel/hung_task: unexport sysctl_hung_task_timeout_secs
Date: Tue, 4 Nov 2025 07:19:20 -0500

This was added by the bcachefs pull requests despite various
objections, and with bcachefs removed is now unused.

This reverts commit 5c3273ec3c6a ("kernel/hung_task.c: export
sysctl_hung_task_timeout_secs").

Link: https://lkml.kernel.org/r/20251104121920.2430568-1-hch@lst.de
Signed-off-by: Christoph Hellwig <hch@lst.de>
Reviewed-by: Lance Yang <lance.yang@linux.dev>
Reviewed-by: Masami Hiramatsu (Google) <mhiramat@kernel.org>
Cc: Kent Overstreet <kent.overstreet@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/hung_task.c |    1 -
 1 file changed, 1 deletion(-)

--- a/kernel/hung_task.c~kernel-hung_task-unexport-sysctl_hung_task_timeout_secs
+++ a/kernel/hung_task.c
@@ -50,7 +50,6 @@ static unsigned long __read_mostly sysct
  * Zero means infinite timeout - no checking done:
  */
 unsigned long __read_mostly sysctl_hung_task_timeout_secs = CONFIG_DEFAULT_HUNG_TASK_TIMEOUT;
-EXPORT_SYMBOL_GPL(sysctl_hung_task_timeout_secs);
 
 /*
  * Zero (default value) means use sysctl_hung_task_timeout_secs:
_
